### DISCUSSION TITLE
Joint rotation for multiple children

### DISCUSSION THREAD

**Discussion by prayshe** [GAMES-105/GAMES-105#8] [Q&A]
I'm confusing about joint rotation and orientation. I could figure it out as for a chain of joints (one joint has at most one child). But what if one parent joint has more than one children, how does it work?

As `child_position = parent_position + parent_orientation * child_offset`, relative positions in between those children are always fixed. Does it mean for a given reference pose (binding pose), some targeting poses couldn't be reached at?

E.g.
`B<----A---->C`,
where A is the parent of B and C, how to let it be
```
   A
  / \
 /   \
B     C
```

**Comment by tragicy** [comment]
In BVH, the defiened child_offset of B and C is different

**Comment by wychah** [comment]
yes,but the child_offset of these joints always have shorter offset,then you can rotate their children to reach target pose.
or they don't have to adjust.like pelvis.
